sdk/storage/azure-storage-blob/azure/storage/blob/aio/_download_async.py

```diff
@@ -14,12 +14,28 @@
 
 from aiohttp import ClientPayloadError
 from azure.core.exceptions import HttpResponseError, ServiceResponseError
+from .._shared.base_client_async import AsyncTransportWrapper
+from .._shared.constants import READ_TIMEOUT
 from .._shared.encryption import decrypt_blob
 from .._shared.request_handlers import validate_and_format_range_headers
 from .._shared.response_handlers import process_storage_error, parse_length_from_content_range
 from .._deserialize import get_page_ranges_result
 from .._download import process_range_and_offset, _ChunkDownloader
 
+
+def get_transfer_timeout(client, request_data_size):
```

Comment:
can we avoid code duplication?

Reply:
the issue with avoiding duplication is the imports. the AsyncTransportWrapper causes import issues on python 2 which is still part of our pipelines/testing framework. I couldn't think of a clean way to check if the transporter object was a transport wrapper object (sync or async). So I was left duplicating in the both the async/sync code